### DISCUSSION TITLE
merge postcss config into tailwindcss config

### DISCUSSION THREAD

**Discussion by alarbada** [tailwindlabs/tailwindcss#12875] [Ideas]
Usually in every tailwind project I've worked with the postcss config file is empty. I don't really like that a css tool needs two config files, one of them being less than 5 lines of code of empty config. Am I alone on this?

**Comment by wongjn** [comment]
For what it's worth, you can "inline" a Tailwind configuration into the PostCSS configuration, like:

```js
module.exports = {
  plugins: {
    tailwindcss: {
      // Tailwind config here.
    },
  },
}
```

**Comment by adamwathan** [comment]
Unfortunately this is more up to the framework or whatever you are using than it is up to us — we can't force Next.js or something for example to automatically load Tailwind without you telling it to load Tailwind in your PostCSS config. I'm hoping to push for more popular frameworks to detect Tailwind though the way [Remix does](https://remix.run/docs/en/main/styling/tailwind) — they automatically set up Tailwind for you if they detect a `tailwind.config.js` file which removes the need for configuring PostCSS like you're talking about.
